packages/api/endpoints/granule-csv.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { Parser } = require('json2csv');
+const router = require('express-promise-router')();
+
+const { Granule } = require('../models');
+
+/**
+ * Builds a CSV file of all granules in the Cumulus DB
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Object} the csv file of granules
+ */
+async function list(req, res) {
+  const granuleModel = new Granule();
+  const allDbGranules = await granuleModel.scan();
+  const fields = ['granuleUr', 'collectionId', 'startDateTime', 'endDateTime'];
+  const granuleArray = [];
+  allDbGranules.Items.forEach((granule) => {
+    const granuleUr = granule.granuleId;
+    const collectionId = granule.collectionId;
+    // Only granules in cmr will only have beginningDateTime/endingDateTime
```

Comment:
It's actually fixed in my PR https://github.com/nasa/cumulus/pull/978.  All granules with .cmr.xml/json file should have these two fields.

Reply:
Also, I think the operator might be interested the `create date time` of the granule.